manageiq-operator/pkg/helpers/httpd_conf/application.conf

```diff
@@ -0,0 +1,38 @@
+# Timeout: The number of seconds before receives and sends time out.
```

Comment:
@carbonin Just an FYI, but this is creating yet another copy (might be required, so maybe we change templates-style to pull from here instead)

Reply:
I think this is probably the one thing I'm okay having a copy of for a while. I really don't see this changing in the time it takes us to move from templates to the operator.